iset.mm

```diff
@@ -129928,27 +129928,27 @@ Norman Megill (2007) section 1.1.3.  Megill then states, "A number of
   althtmldef "-1-1->" as
     '&ndash;<FONT SIZE=-2 FACE=sans-serif>1-1</FONT>&rarr;';
   latexdef "-1-1->" as
-    "\raisebox{.5ex}{${\textstyle{\:}_{\mbox{\footnotesize\rm 1" +
-    "\tt -\rm 1}}}\atop{\textstyle{" +
-    "\longrightarrow}\atop{\textstyle{}^{\mbox{\footnotesize\rm {\ }}}}}$}";
+    "\raisebox{.5ex}{${\textstyle{\:}_{\mbox{\footnotesize\mathrm{1}" +
+    "\tt -\mathrm{1}}}}\atop{\textstyle{" +
+    "\longrightarrow}\atop{\textstyle{}^{\mbox{\footnotesize\mathrm{\ }}}}}$}";
```

Comment:
> \mathrm{\ }

not sure about that but this overly complicated latexdef will hopefully soon go away, see #3054

Reply:
Well, `\mathrm{\ }`  does have an effect - it creates horizontal space like a Roman-style space. If we don't need it, let's remove it, but I presumed it was put in for a reason. Let's make removing that a separate proposal (as discussed).